.changeset/itchy-apes-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+'@twilio-paste/website': minor
+---
+
+add scroll to top link to website tokens page
```

Comment:
Yeah, potentially confusing in this monorepo. We don't tend to version our apps or websites because they don't really have published releases. Every commit is a release for those.

We used to, but then we stopped. I don't think either way is wrong, just that there is no requirement for it right now.

Anything that is marked as a private package I think follows the same pattern. Not versioned.